ci/hardware_tests/default.nix

```diff
@@ -0,0 +1,15 @@
+with import <nixpkgs> {};
+
+stdenv.mkDerivation rec {
+  name = "trezor-firmware-hardware-tests";
+  buildInputs = [
+    uhubctl
+    ffmpeg
+    pipenv
+    scons  # needed for our pipenv to sync
```

Comment:
I think this is not necessary. (And I would remove `scons` from the main `shell.nix` as well).

We know we discussed this via Meet, but it seems to me we omitted something and now this works for me. (When I remove `scons` from the main `shell.nix` I can use scons installed into the virtualenv without any problem). Let's another call if that still causes you the troubles.

Reply:
This sequence works for you?
```
rm -rf ~/.local/share/virtualenvs/firmware-*
cd ci/hardware_tests/  
vi default.nix                # delete scons here
nix-shell --pure
pipenv sync
```

Reply:
Yes, this indeed works for me.

Reply:
I would waste your time on fixing this issue and if adding scons works for you, let's keep it that way (with the comment) and we will revisit this later (and we can also remove `scons` from the main `shell.nix` at this point).

Reply:
The problem is I am curious now :)). I have pushed it to CI, because that's how I've figured it out in the first place. https://gitlab.com/satoshilabs/trezor/trezor-firmware/pipelines/133768862

Reply:
@prusnak https://gitlab.com/satoshilabs/trezor/trezor-firmware/-/jobs/502417279 . But I agree it is weird and adding `scons` si some weird hotfix anyway.

Reply:
Ah, I know what is causing the issue. `StrictVersion` used in the python-fido2 package cannot handle weird versions such as `45.2.0.post20200310`. This issue has been fixed in the upstream already (https://github.com/Yubico/python-fido2/commit/246d6f59d5e60bfbe9a3c9fd04fbb7df7d02c589). We haven't picked up the change, because we are stuck with python-fido2 0.7.3 because of this issue: https://github.com/solokeys/fido2-tests/issues/27

Luckily, this has been fixed in the fido2-tests repo, so I updated our fork of fido2-tests.

Unluckily, I hit the https://github.com/trezor/trezor-firmware/issues/934 while fixing it. So this needs to be fixed first.

Reply:
Give me some time before merging this PR please and we'll figure this out completely.

Reply:
Master is green in the CI, so my take is that if you rebase this branch of top of master, removing `scons` from ` ci/hardware_tests/default.nix` it should work.

Reply:
Done. Looks good!